static/src/javascripts/projects/commercial/modules/header-bidding/prebid/price-config.ts

```diff
@@ -35,3 +35,56 @@ export const criteoPriceGranularity: PrebidPriceGranularity = {
 		},
 	],
 };
+
+/**
+ * Compute the price granularity for Ozone based on the width and height of the slot
+ */
+export const ozonePriceGranularity = (
+	width: number,
+	height: number,
+): PrebidPriceGranularity | undefined => {
+	switch ([width, height].join(',')) {
+		case '160,600':
```

Comment:
Could these be the imported `ad-sizes` from commercial-core?

Also fine as a follow up too for when we revisit ad-sizes in which case a TODO might be handy?

Reply:
Good shout. I was wary about doing this because JS switches aren't smart enough to pattern match on non-primitive values, but I think it's worth foregoing the switch in this instance.